source/common/http/conn_manager_impl.cc

```diff
@@ -857,10 +857,19 @@ void ConnectionManagerImpl::ActiveStream::decodeData(ActiveStreamDecoderFilter*
   if (!filter) {
     entry = decoder_filters_.begin();
   } else {
-    entry = std::next(filter->entry());
+    if ((*(filter->entry()))->iterate_from_current_filter_) {
```

Comment:
Is there a way to remove `iterate_from_current_filter_` to make the logic simpler? I'm wondering if we check the current filter's stop state, and if stopped, just do the common data after stop logic?

Reply:
Good call! Fixed.